components/contribution-flow/index.js

```diff
@@ -529,7 +552,7 @@ class ContributionFlow extends React.Component {
     }
 
     // Hide step payment if using a free tier with fixed price
-    if (!noPaymentRequired) {
+    if (!noPaymentRequired && !isCrypto) {
```

Comment:
The code comment was explaining the rationale of the condition. We add `isCrypto` and we don't explain why.

Please add a comment.

Reply:
Done. 👍🏽